### PR TITLE
add policy-option specific components only for HOME product line

### PR DESCRIPTION
**Pull request by mariodavid**
This PR creates different react components for the different policy options (glassOption, bikeOption etc.) and displays them in the case of a product with a HOME productLine.

It displays the options always. In case the option is not included in the policy, it has a corresponding "disabled" css class, which can be used to style it accordingly.

### CODE REVIEW

File: src/policy/PolicyDetails.js [hunk 1]
```diff
@@ -58,33 +59,137 @@ const PolicyDetailItem = (props) => {
...
+    else {
+        wrapperClassName += " disabled";
+        return (
+            <div className={wrapperClassName}>
+                <h3>{name}</h3>
+                <img src={imgSource}/>
+                <p>{description}</p>
+            </div>
+        );
+    }
```
Comment (ulich) [inline src/policy/PolicyDetails.js]: quite a lot of code duplication just for the `Individual Coverage` thing. Maybe that can be improved by creating a property where you can pass in the Individual Coverage div

Comment (mariodavid) [reply src/policy/PolicyDetails.js]: good idea. Changed it through passing child elements.

File: src/policy/Money.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+export default function Money({value}) {
+    return (
+        <NumberFormat value={value}
+                         displayType={'text'}
+                         thousandSeparator={'.'}
+                         suffix={' €'}
+                         decimalSeparator={','}/>
+    )
+}
```
Comment (ulich) [inline src/policy/Money.js]: 💋

File: src/policy/PolicyDetails.js (hunk 1 above)
Comment (ulich) [reply src/policy/PolicyDetails.js]: a lot better :)

**Pull request merged**